core/domain/opportunity_services.py

```diff
@@ -498,27 +463,50 @@ def get_translation_opportunities(language_code, topic_name, cursor):
         .ExplorationOpportunitySummaryModel.get_all_translation_opportunities(
             page_size, cursor, language_code, topic_name))
     opportunities = []
-    suggestion_ids = []
-    opportunity_ids = []
+    exp_id_to_in_review_count = (
+        _build_exp_id_to_translation_suggestion_in_review_count(
+            exp_opportunity_summary_models, language_code))
```

Comment:
If I'm reading the code right, all you need in the private method is the IDs, right -- and nothing else from the models? If so, I would suggest passing just a list of model IDs around. 

In general, try not to pass datastore models around the codebase, they contain lots of methods like put() etc. that we don't want the domain layer to call. As a general rule, it's better to avoid the use of models as much as possible -- convert them to domain objects or IDs etc. at the first opportunity.